times_table.js

```diff
@@ -0,0 +1,32 @@
+const pry = require('pryjs')
+
+const printFormat = function(set) {
+    let printable = ""
+    set.forEach(function(row) {
+        row.forEach(function(number) {
+            printable += `${number} `
+        })
+        printable += "\n"
+    })
+    return printable
+
+}
+
+const pushRow = function(i, width) {
+    let row = []
+    for (var j = 1; j <= width; j++) {
+        row.push(i * j)
+    }
+    return row
+}
+
+const timesTable = function(width) {
+    var set = []
```

Comment:
Stick with `const` over `var`. Only switch to `let` from `const` if you have to.